Common/src/containers/CFileReaderLUT.cpp

```diff
@@ -238,15 +240,67 @@ void CFileReaderLUT::ReadRawDRG(const string& file_name) {
 
 }
 
-string CFileReaderLUT::SkipToFlag(ifstream* file_stream, const string& flag) {
-  string line;
-  getline(*file_stream, line);
+/**
+ * @brief Searches for the position of flag in file_stream and 
+ *        sets the stream position of file_stream to that position.
+ * 
+ * @param file_stream 
+ * @param current_line 
+ * @param flag 
+ */
```

Comment:
Please use the style we have in other parts of the code `\param[in]` `\brief` and there is a `!` missing somewhere.